utils/service_discovery/sd_docker_backend.py

```diff
@@ -106,6 +106,7 @@ def __init__(self, agentConfig):
         self.VAR_MAPPING = {
             'host': self._get_host_address,
             'port': self._get_port,
+            'containername': self._get_container_name,
```

Comment:
Could you make it part of `%%tags%%` instead please? in `_get_additional_tags` you can call `_get_container_name` and add the `container_name:` prefix.